AutomatedTesting/Gem/PythonTests/editor/TestSuite_Main_Optimized.py

```diff
@@ -0,0 +1,75 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+import pytest
+
+import ly_test_tools.environment.file_system as file_system
+from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorSharedTest, EditorParallelTest, EditorTestSuite
+
+
+@pytest.mark.xfail(reason="Optimized tests are experimental, we will enable xfail and monitor them temporarily.")
+@pytest.mark.SUITE_main
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+class TestAutomationNoAutoTestMode(EditorTestSuite):
+
+    # Disable -autotest_mode and -BatchMode. Tests cannot run in -BatchMode due to UI interactions, and these tests
+    # interact with modal dialogs
+    global_extra_cmdline_args = []
+
+    class test_BasicEditorWorkflows_LevelEntityComponentCRUD(EditorSingleTest):
+        # Custom teardown to remove slice asset created during test
+        def teardown(self, request, workspace, editor, editor_test_results, launcher_platform):
+            file_system.delete([os.path.join(workspace.paths.engine_root(), "AutomatedTesting", "Levels", "tmp_level")],
+                               True, True)
+        from .EditorScripts import BasicEditorWorkflows_LevelEntityComponentCRUD as test_module
```

Comment:
Not for this PR, but you can avoid creating and then deleting a level.
You can use an already existing empty level, open it and just make sure you don't save it when you finish the test